instrumentation/grape/lib/opentelemetry/instrumentation/grape/subscriber.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module Grape
+      # Manages all subscriptions, both for custom subscribers and built-in notifications
+      class Subscriber
+        class << self
+          # Subscribe to all custom and built-in notifications (except those specified in the :ignored_events configs)
+          def subscribe
+            subscribe_to_custom_notifications
+            subscribe_to_built_in_notifications
+          end
+
+          private
+
+          SUBSCRIPTIONS = {
+            endpoint_render: 'endpoint_render.grape',
+            endpoint_run_filters: 'endpoint_run_filters.grape',
+            format_response: 'format_response.grape'
+          }.freeze
+
+          CUSTOM_SUBSCRIPTIONS = {
```

Comment:
Proposal. Feel free to decline if this is too verbose etc.

```suggestion
          # ActiveSupport::Notifications to be subscribed to that implement a
          # ActiveSupport::Notifications::Fanout::Subscribers::Evented subscriber
          # 
          # Reference: https://github.com/rails/rails/blob/05cb63abdaf6101e6c8fb43119e2c0d08e543c28/activesupport/lib/active_support/notifications/fanout.rb#L320-L322
          CUSTOM_SUBSCRIPTIONS = {
```

Reply:
Added it here in a shortened form but also added it to the `CustomSubscribers` module ❤️